pkgs/development/tools/swiftformat/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, lib, fetchFromGitHub, runCommand }:
+
+# This derivation is impure: it relies on an Xcode toolchain being installed
+# and available in the expected place. The values of buildSymlinks,
+# sandboxProfile, and hydraPlatforms are copied pretty directly from the MacVim
+# derivation, which is also impure.
+
+let
+  buildSymlinks = runCommand "swiftformat-build-symlinks" {} ''
+    mkdir -p $out/bin
+    ln -s /usr/bin/xcodebuild $out/bin
```

Comment:
Do we actually need this? You're not invoking some Makefile that references `xcodebuild` or anything like that, you're just invoking `xcodebuild` directly in `buildPhase`, so I would imagine you could just invoke `/usr/bin/xcodebuild` instead.

Reply:
Indeed you’re right. I’ve force-pushed a new commit that removes that.